src/renderer/api/kube-watch-api.ts

```diff
@@ -127,45 +128,66 @@ export class KubeWatchApi {
     };
   }
 
-  subscribeStores(stores: KubeObjectStore[], options: IKubeWatchSubscribeStoreOptions = {}): () => void {
-    const { preload = true, waitUntilLoaded = true, cacheLoading = false } = options;
+  preloadStores(stores: KubeObjectStore[], { loadOnce = false } = {}) {
     const limitRequests = plimit(1); // load stores one by one to allow quick skipping when fast clicking btw pages
```

Comment:
If you want fast skipping then wouldn't be better to support passing in an `AbortController`?

Reply:
No, it's not related to watch-api request itself... (it's preloading stores)

Reply:
Yes I understand that this function is for preloading stores. But I don't understand what that has to do with this call? The comment implies that we only do one request at a time to allow for quick skipping. Which I think implies stopping the previous preload and starting another one.

If the above is true then that is a perfect use case for something like `AbortController`

Reply:
- `AbortController` aborts only request `POST /api/watch` (watch-route.ts)
- `plimit` aborts other k8s requests, e.g `GET /api/v1/pods`

Do you suggest to provide extra `AbortController` to k8s requests to abort them all at once?

Reply:
Yes I do since `plimit` doesn't actually abort the requests.

Reply:
Yes, but since requests are made one by one not so critical (only last active load will be aborted).
Could be improved anyway in follow-up PR.

Reply:
I meant in conjunction with raising the `plimit` value